### DISCUSSION TITLE
Resolve Conflict button is disabled

### DISCUSSION THREAD

**Discussion by buddhikac96** [community/community#22431] [New to GitHub]
<p>I recently created a pull request to a branch of a repository (which I have no write access to) and there are some conflicts. But the resolve conflict button is disabled due to the complex. Now how can I resolve this?</p>
<p>I guess I have to resolve this locally and push to the same PR. But I am not sure what to do. More descriptive answer will truly help me since I am not a pro for GitHub.</p>
<p>Thank you!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There are two common ways to resolve this, and yes, both are local. Either way you need to have the repository you want to merge into configured as a remote in your local repository, I’ll assume that remote is called <code>upstream</code>.</p>
<ol>
<li>Rebase your branch on the upstream branch you want to merge into. This will recreate each commit on the branch starting from the upstream branch, and you’ll have to resolve conflicts on each commit.</li>
</ol>
<pre><code class="lang-auto">git checkout your-branch
git rebase upstream/master
</code></pre>
<ol start="2">
<li>Merge the upstream branch into your local branch. This does not modify your existing commits, and instead adds the new upstream commits plus a merge commit. You’ll have to resolve conflicts during the merge.</li>
</ol>
<pre><code class="lang-auto">git checkout your-branch
git merge upstream/master
</code></pre>
<p>When done, push to the same branch you started the pull request from. With the rebase option you’ll have to force-push, so double check the result is correct. The pull request should be updated automatically when you push.</p>

**Comment by buddhikac96** [comment]
<p>Thanks a lot. This helped!!!</p>

**Comment by Norman-86** [comment]
<p>This was quite helpful. I was stuck with a deactivated “Resolve Conflicts” tab.</p>
